index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="utf-8">
+        <title>Assignment HTML</title>
```

Comment:
Even though this is an assignemnt, think of it as your personal project. give it a title that you'd like to see in a the browser tab 😉

Reply:
You're absolutely correct. :satisfied: